README.md

```diff
@@ -311,14 +307,11 @@ skate('my-element', {
       // a custom value, specify a getter.
       set: function (newValue, oldValue) {},
 
-      // A function that coerces the value to another value. You can specify
-      // any function you want here. The return value is then stored internally
-      // and passed as `newValue` in the setter.
-      type: Boolean,
-
-      // This will be used as the initial value for the property. If you specify
-      // a function then it will be invoked and the return value will be used.
-      value: 'initial value'
+      // A function that gets called before `set()`. It's up to you what you do
+      // here. You can log, warn, or throw an exception if an unacceptable value
+      // is detected. By default this does nothing and the return value is not
+      // used.
```

Comment:
> // … By default this does nothing and the return value is not
> // used.

This is no longer accurate, right? `src/lifecycle/property.js` sets `newValue` to the result of this if you've provided it.